internal/cluster/kube/deployment_test.go

```diff
@@ -0,0 +1,78 @@
+package kube
+
+import (
+	"github.com/stretchr/testify/assert"
+	"k8s.io/client-go/kubernetes/fake"
+	"kube-proxless/internal/cluster"
+	"kube-proxless/internal/utils"
+	"testing"
+)
+
+func Test_scaleUpDeployment(t *testing.T) {
+	TestClusterClient_ScaleUpDeployment(t)
+}
+
+func Test_waitForDeploymentAvailable(t *testing.T) {
+	clientSet := fake.NewSimpleClientset()
+
+	timeout := 1
+
+	// error - deployment is not in kubernetes
+	assert.Error(t, waitForDeploymentAvailable(clientSet, dummyProxlessName, dummyNamespaceName, timeout))
+
+	helper_createNamespace(t, clientSet)
```

Comment:
comment - maybe helper.createNamespace and create a package for the funcs?

Reply:
unfortunately, I already tried and I cannot import `test` packages. 😭 unless u want to define a `non test` package?